Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 gem 'fastlane'
 gem 'snapshot'
+gem 'cocoapods', '= 1.10.0'
```

Comment:
It would be useful to test against new minor and patch releases as they come out, since the purpose of the CocoaPods integration test is to verify that a normal workflow would install correctly. v1.10.0 is an important baseline for its XCFramework support, but aside from that we don’t seem to have any other technical constraints.

Reply:
Makes sense, updated `Gemfile`.